session_items.py

```diff
@@ -1,73 +1,111 @@
 from flask import session
+import os
+import requests
+
+from Task import Task, Status
 
 _DEFAULT_ITEMS = [
     { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
     { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
 ]
 
+BOARD_ID = os.environ.get('BOARD_ID')
+TRELLO_KEY = os.environ.get("TRELLO_KEY")
+TRELLO_TOKEN = os.environ.get("TRELLO_TOKEN")
+
+TODO_LIST_ID = os.environ.get("TODO_LIST_ID")
```

Comment:
These two could be inferred from the list names

Reply:
Can a have a method which runs whenever the app is launched, which retrieves and stores them only if it doesn't already have them? sort of like an init of the whole app? It seems silly to retrieve them separately every time, and seems like it'd clutter the code check for them on every method and fetch if we don't have.

Reply:
I'd advocate keeping the webserver as stateless as possible at this point. Later on we can introduce database and caching layers to work around this, but I'd argue that's overkill for this exercise.